src/middleware/withAuthorizedSpace.js

```diff
@@ -75,14 +76,22 @@ function extractSpaceDID (space) {
 export function withAuthorizedSpace (handler) {
   return async (request, env, ctx) => {
     const { locator, dataCid } = ctx
+
+    console.log(`[withAuthorizedSpace] Locating content: ${dataCid}`)
```

Comment:
There are a lot of console.log calls here -- is this intentional or an artifact of debugging -- ok either way, just checking